js_modules/dagster-ui/packages/app-oss/src/publicPath.ts

```diff
@@ -1,16 +0,0 @@
-/**
- * This file must be the first import in the application entry file.
- * https://webpack.js.org/guides/public-path/#on-the-fly
- */
-
-import {extractInitializationData} from './extractInitializationData';
-
-const extracted = extractInitializationData();
-
-// Set the webpack path prefix based on DOM value. This will be used
-// for dynamically loaded bundles.
-if (typeof extracted.pathPrefix === 'string') {
```

Comment:
oh hmm.... next.js expects the `pathPrefix` to be set at build time so I removed this file, but I need to figure out a way to make this work otherwise.... 🤔